packages/dd-trace/src/config.js

```diff
@@ -673,6 +674,7 @@ class Config {
       DD_TRACE_CLIENT_IP_HEADER,
       DD_TRACE_DYNAMODB_TABLE_PRIMARY_KEYS,
       DD_TRACE_ENABLED,
+      DD_TRACE_EXPERIMENTAL_ENABLED,
```

Comment:
This seems to be missing a word? What experimental feature is enabled?

Reply:
Currently this is a catch all flag to enable all experimental features hidden behind the flag, but there is a proposal out right now to evolve this to also allow for targeting  of which features to enable

Reply:
More so for allowing breaking features before a major release